src/topology/algebra/field.lean

```diff
@@ -141,3 +142,14 @@ def affine_homeomorph (a b : 𝕜) (h : a ≠ 0) : 𝕜 ≃ₜ 𝕜 :=
   right_inv := λ y, by { simp [mul_div_cancel' _ h], }, }
 
 end affine_homeomorph
+
+section local_extr
+
+variables {α β : Type*} [topological_space α] [linear_ordered_semifield β] {a : α}
+open_locale topological_space
+
+lemma is_local_min.inv {f : α → β} {a : α} (h1 : is_local_min f a) (h2 : ∀ᶠ z in 𝓝 a, 0 < f z) :
```

Comment:
This should be true with `∀ᶠ z in 𝓝 a, 0 ≠ f z` instead of `∀ᶠ z in 𝓝 a, 0 < f z` right? Maybe with stronger assumptions like `linear_ordered_field`, in which case we should also keep your version, but could you add the `≠` one while you're at it?

Reply:
That would be true with an additional step that all the values have the same sign (otherwise `inv` is not monotone) and this requires some continuity assumption somewhere for `f`.

Reply:
Oh sorry I thought `f` was continuous. Forget my comment then.

Reply:
Sure if you want to do it that would be nice

Reply:
So, `continuous_at f a` and `f a \ne 0` then, that should be enough (continuity is certainly needed, and these two imply eventually \ne 0. I will add that to my todo-list for this PR, and do it if it boils down to a few lines.

Reply:
OK so finally that will be for another PR if I need it later...

Reply:
Okay, no problem!